roles/cluster/cluster_deploy_kepler/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Grab kepler-operator source
+  command:
+    git clone https://github.com/sustainable-computing-io/kepler-operator "{{ artifact_extra_logs_dir }}/kepler"
+
+- name: Deploy kepler-operator
+  shell:
+    cd "{{ artifact_extra_logs_dir }}/kepler"; make deploy OPERATOR_IMG=quay.io/sustainable_computing_io/kepler-operator:latest
+
+- name: Deploy kepler exporters
+  shell:
+    oc apply -k "{{ artifact_extra_logs_dir }}/kepler/config/samples/"
+
+- name: Ensure kepler was deployed
+  shell:
+    oc get -n openshift-kepler-operator svc -oname
```

Comment:
why is it deploying in `openshift-kepler-operator` ? 🤔 
_this_ is the reason why I was seeing a different behavior, `openshift-` namespaces are trusted by default, they're not "user" namepsaces
(but I don't see which command tells it that it's an OpenShift cluster 🙃 )